src/pages/pages.scss

```diff
@@ -69,9 +69,9 @@
   }
 }
 
-.usingCustomTelematicsDevices, .javascriptExamples {
```

Comment:
please get rid of all of this. we should have one class for tables and one class for images